Vagrantfile

```diff
@@ -55,20 +56,17 @@ Vagrant.configure('2') do |config|
   end
 
   config.vm.provision 'shell', inline: <<-SHELL
-    yum -y install yum-plugin-copr epel-release http://download.zfsonlinux.org/epel/zfs-release.el7_5.noarch.rpm
+    yum -y install yum-plugin-copr epel-release http://download.zfsonlinux.org/epel/zfs-release.el7_6.noarch.rpm
+    yum-config-manager --disable zfs
+    yum-config-manager --enable zfs-kmod
     yum -y copr enable alonid/llvm-5.0.0
-    yum -y install clang-5.0.0 zfs libzfs2-devel --nogpgcheck
-    modprobe zfs
+    yum -y install clang-5.0.0 zfs libzfs2-devel cargo --nogpgcheck
     genhostid
```

Comment:
This may want to be `zgenhostid` (provided by zfs instead of built-in from rhel)

Reply:
Ok, I'll try it